tests/plugins/widget/integration/table/manual/widgetintablewithtableselection.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: 1027, 4.8.0, widget
```

Comment:
This test could be ignored for IE8-10 as `tableselection` is not supported there so it will be exactly the same as `.../manual/widgetintable` test.